### PR TITLE
Make install script compatible with Python2 and 3

### PR DESCRIPTION
**Pull request by Duhemm**
Translated using `futurize`.

@valencik I don't know much about Python, is this the way to go?

### CODE REVIEW

File: bin/install.py [hunk 1]
```diff
@@ -55,10 +58,10 @@
 
 def download_and_install(url, target):
     try:
-        urllib.urlretrieve(url, target)
-        os.chmod(target, 0755)
+        urllib.request.urlretrieve(url, target)
+        os.chmod(target, 0o755)
```
Comment (jvican) [inline bin/install.py]: `0o`? You sure?

Comment (Duhemm) [reply bin/install.py]: Yes (that's generated code, and it runs fine in the Python REPL)

Comment (jvican) [reply bin/install.py]: Let the Python REPL guide us!

**Comment by valencik** [comment]
Oh wow, I didn't know `futurize` could do this without having a new dependency. Very nice!

**Pull request merged**